javascript/javascript3/week1/giphyApp/giphyApp.js

```diff
@@ -0,0 +1,45 @@
+const btnGetGif = document.getElementById('btnGetGif');
+const numberInput = document.getElementById('numberInput');
+const displayResult = document.getElementById("searchResult");
```

Comment:
`const displayResult = document.getElementById("searchInput");`
 See index.html where`<input type="text" id="searchInput" placeholder="Search for gif">`